app/src/test/java/com/github/siela1915/dishdelish/AutocompleteTest.java

```diff
@@ -1,21 +1,102 @@
 package com.github.siela1915.dishdelish;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+
+import com.github.siela1915.bootcamp.AutocompleteApi.ApiResponse;
+import com.github.siela1915.bootcamp.AutocompleteApi.ApiService;
 import com.github.siela1915.bootcamp.AutocompleteApi.IngredientAutocomplete;
 
+import org.checkerframework.checker.units.qual.A;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
 
-import static org.junit.Assert.*;
-
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
+import okhttp3.Request;
+import okio.Timeout;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
 public class AutocompleteTest {
-    IngredientAutocomplete fetcher = new IngredientAutocomplete();
+    private final String apiKey = "44a82829c64d4202a18b887e47e76bdd";
```

Comment:
Having api key in public GitHub repository might be a bad idea, try to move it to an environment variable or something (I think there was some discussion on SDP discord concerning this)